_posts/2020-10-04-newpipe-0.20.0.md

```diff
@@ -1,154 +1,176 @@
 ---
 layout: post
-title: "NewPipe 0.20.0 released: A new major version, a new unified player, a new notification experience... A lot is new this time!"
+title: "NewPipe 0.20.0 released: Major version, new unified player, new notifications. More things."
 short: "NewPipe 0.20.0 released"
 date: 2020-10-04 05:00:00 +0100
 categories: [pinned, release]
-author: Opusforlife & Poolitzer
+author: Opusforlife, Poolitzer and Allan Nordhøy
 image: newpipe
 excerpt_separator: <!-- more -->
 ---
 
-Hey there! Guess who is back? We are. Admittedly, we're quite a bit late with this release. But there is a _very_ good reason for that. Are you ready for it? Sit down, grab some popcorn and a beverage of your choice. This is going to be long.
+Hey there! Back admittedly a bit late with this release, but for _very_ good reason. Ready? Sit down, grab the virtual popcorn and some hydration.
 <!-- more -->
 
 ## A new unified player
 
-"What in the name of the coronavirus is the _unified player_?" you ask. Well, first of all, that's a very weird expression. Where did you even get that one from... - you know what? Save it. We don't want to know. You just keep on using that, we'll keep on talking about the new player.
```

Comment:
I don't understand the corona joke. Breaking the fourth wall in what is now different tenses (under different pretenses) doesn't work. Telling the user to "save it" and "we don't want to know" adds insult to injury. There is no landscape created to pull this off, it is at face value a construction that falls on itself. There has been nothing said about the unified player at all, at length I might add.

Reply:
"In the name of God" - A phrase of exasperation used to emphasize a question or statement. "What in the name of God are you doing in here?"

That's the corona joke.

Can you please point out where the different tense is?

The user didn't come up with that expression, we did. The user knows this, so why would they take offence to us making fun of something they didn't even say? We are deliberately attacking a strawman here. That _is_ the joke.

Reply:
Talking with the user, talking to the user, asking rhetorical or hypothetical questions, under a headline that seems fairly normal. OK, there is a functional setup for a joke, but it doesn't tie together. Am I still missing something?

Reply:
Not sure what the headline has to do with anything, but I assure you, the joke works.

> Talking with the user, talking to the user,

*Talking _for_ the user, talking to the user